src/content/en/updates/2017/07/upcoming-regexp-features.md

```diff
@@ -0,0 +1,213 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: This update discusses upcoming features in JavaScript RegExps.
+
+{# wf_updated_on: 2017-07-10 #}
+{# wf_published_on: 2017-07-10 #}
+{# wf_tags: javascript #}
```

Comment:
Add regex as a tag. Additionally, add it to the [commonTags.json file](https://github.com/google/WebFundamentals/blob/master/src/data/commonTags.json).